### ISSUE TITLE
Homepage email sign up hint text does not pass contrast ratios

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23104]
## Open questions
- How do we want to handle this for the home page?
  - [Initial discussion on slack 🔒](https://dsva.slack.com/archives/C52CL1PKQ/p1764949555842759)
    - Currently considering removing the character count hint text altogether for this use case. Is that the path forward here?
    - How do we want to handle an overly long email? Error message?

----
## Describe the defect
Hint text for the email sign up block on the [homepage](https://www.va.gov/) does not pass contrast ratios with the blue background.

Defect Severity: 2, Serious. Should be fixed in 1-2 sprints post-launch.
Severity rationale: This issue is occurring on the homepage of VA.gov. Users with low vision, color blindness or other vision issues may not be able to see this text at all

<img width="1144" height="643" alt="Image" src="https://github.com/user-attachments/assets/2f3cba46-9d1e-41d5-a2e9-df455c627921" />

## To Reproduce
1.  Go to the [VA.gov homepage](https://www.va.gov/)
2. Scroll to the bottom of the page, to the Email section
3. Notice the character counter
4. Run axeDev tools ([installation info](https://docs.deque.com/devtools-for-web/4/en/devtools-extension#installation))
5. See the color contrast violation that this hint text color does not pass color contrast ratio requirements against the blue background.

## AC / Expected behavior

- [ ] TBD

 ## Additional context
- Related DST ticket: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5082
